vite.config.js

```diff
@@ -39,7 +39,7 @@ export default defineConfig({
       css: true
     }),
     vueI18n({
-      include: path.resolve(__dirname, 'src/assets/translations.ts'),
+      include: path.resolve(__dirname, './src/locales/**.json'),
```

Comment:
`@intlify/vite-plugin-vue-i18n` cannot use `typescript` files as locale resources.

Reply:
Hi there, thanks for quick response and fix